site2/docs/reference-cli-tools.md

```diff
@@ -174,10 +174,15 @@ Options
 |`-ub` , `--broker-service-url`|The broker service URL for the new cluster||
 |`-tb` , `--broker-service-url-tls`|The broker service URL for the new cluster with TLS encryption||
 |`-c` , `--cluster`|Cluster name||
-|`--configuration-store`|The configuration store quorum connection string||
+|`-cs` , `--configuration-store`|The configuration store quorum connection string||
+|`--existing-bk-metadata-service-uri`|The metadata service uri of existing BookKeeper cluster you want to use||
+|`-h` , `--help`|Cluster name|false|
+|`--initial-num-stream-storage-containers`|Num storage containers of BookKeeper stream storage|16|
+|`--initial-num-transaction-coordinators`|Num transaction coordinators will assigned in cluster|16|
 |`-uw` , `--web-service-url`|The web service URL for the new cluster||
 |`-tw` , `--web-service-url-tls`|The web service URL for the new cluster with TLS encryption||
 |`-zk` , `--zookeeper`|The local ZooKeeper quorum connection string||
+|`--zookeeper-session-timeout-ms`|Local zookeeper session timeout ms|30000|
```

Comment:
```suggestion
|`--zookeeper-session-timeout-ms`|Local ZooKeeper session timeout ms|30000|
```

Reply:
`ms` is the time unit which is short for `millisecond`.

Reply:
Thanks for your clarification. Maybe we can add it in the description, so it will be clear to users.